examples/zoomable-chart/highlight.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {ScaleUtils, AbstractSeries} from 'react-vis';
+
+export default class Highlight extends AbstractSeries {
+
+  static displayName = 'HighlightOverlay';
+  static defaultProps = {
+    allow: 'x',
+    color: 'rgb(77, 182, 172)',
+    opacity: 0.3
+  };
+
+  constructor(props) {
```

Comment:
i think it would be better to set the state directly without using the props?  eg

```javascript
{
 +      drawing: false,
 +      drawArea: {top: 0, right: 0, bottom: 0, left: 0},
 +      startLoc: 0
 +    }
```